src/service/auth-services.jsx

```diff
@@ -2,18 +2,33 @@ import { tokenKey } from "../config";
 import collectionClient from "./collection-client";
 
 export async function login(credentials) {
-	const {token, ...user} = await collectionClient("/login", {
-	body: credentials 
-	});
+  const { ...user } = await collectionClient("/login", {
+    body: credentials,
+  });
```

Comment:
If it would only return a user object there is no need to use object destructuring:

```suggestion
  const user = await collectionClient("/login", {
    body: credentials,
  });
```